modules/ROOT/pages/create-an-api-configure-service-mesh-CRD.adoc

```diff
@@ -100,6 +108,120 @@ You must have the required permissions to create the Kubernetes secret.
+  clusterServicePlanExternalName: service-mesh
+  parameters:
+    environmentId: _environment ID_ # example: 7cea45c6-3237-464d-bd34-3122c4abde29
+    groupId: _group ID_ # Optional - example: 350b8164-e809-4091-91f5-079fb30e5a1e
+    assetId: _asset ID_ # example: orders-api
+    version: _version_ # example: 0.0.1
+    instanceLabel: _instance label_ # example: public
+    tags:
+      apiSpec:
+      - _tag in Exchange asset_ # example: orders
+      apiInstance:
+      - _tag in API Manager instance_ # example: orders
+    clientId: _ID of the Anypoint Platform connected app_
+    clientSecret: _Secret of the Anypoint Platform connected app_
+----
+
+[[create-api-CRD-ConnAps]]
+== Create an API Using Anypoint Service Mesh With Client Credentials
+
+Use the procedure in this section to automatically create an API using Anypoint Service Mesh using your Conncted App client credentials. Before you create the API, you must complete the task prerequisite.
+
+*Task Prerequisite*
+
+Before you begin, ensure that you:
+
+* You xref:obtain-connected-apps-credentials.adoc[Obtain the client ID and client secret credentials] from  Connected Apps
+* The client credentials has the:
+** Exchange Contributor permission to publish the API to Exchange
+** Manage API Configuration to manage the API in API Manager
+
+To create an API in Anypoint Platform using Anypoint Service Mesh:
+
+. Create your api definition file and save it with a '.yaml' file extension, for example `test-api.yaml`:
++
+[source,text,linenums]
+----
+apiVersion: servicecatalog.k8s.io/v1beta1
+kind: ServiceInstance
+metadata:
+  name: _Kubernetes API resource name_ # example: orders-api
+  namespace: _namespace name_ # example: default
+  labels: 
+    service-mesh.mulesoft.com/kind: api
+spec:
+  clusterServiceClassExternalName: anypoint-platform-api-instance
+  clusterServicePlanExternalName: service-mesh
+  parameters:
+    environmentId: _environment ID_ # example: 7cea45c6-3237-464d-bd34-3122c4abde29
+    groupId: _group ID_ # Optional - example: 350b8164-e809-4091-91f5-079fb30e5a1e
+    assetId: _asset ID_ # example: orders-api
+    version: _version_ # example: 0.0.1
+    instanceLabel: _instance label_ # example: public
+    tags:
+      apiSpec:
+      - _tag in Exchange asset_ # example: orders
+      apiInstance:
+      - _tag in API Manager instance_ # example: orders
+    user: _Username of the Anypoint Platform user_
+    password: _Password of the Anypoint Platform user_
+----
+. Apply the CRD file using the following command:
+`kubectl apply -f test-api.yaml`
+. Verify the API status using the following command:
+`asmctl api list`
++
+The API that was created is displayed.
+. Optionally, you can verify that an asset is created on Exchange and an API is created on API Manager.
+
+== Reference Kubernetes Secret
+
+Anypoint Service Mesh supports the use of Kubernetes secrets to store and manage sensitive information, such as your user credentials.
```

Comment:
Please, change the text of this sentence for:
```suggestion
Anypoint Service Mesh supports the use of Kubernetes secrets to store and manage sensitive information, such as your Connected App credentials.
```

Reply:
Done! Thanks!